app/views/partials/foyer/logement.html

```diff
@@ -102,14 +91,18 @@
           </label>
           <div class="col-sm-6">
             <div class="input-group">
-              <input type="number" class="form-control text-right" id="loyer" name="loyer" ng-model="logement.loyer" required min="1">
+              <input type="number" class="form-control text-right" id="loyer" name="loyer" ng-model="logement.loyer">
```

Comment:
`min=0` would have been ok to keep (+ add `step="100" max="9999"`, for example). The reason is, as soon as you set `type="number"`, the browser will display arrows to increment and decrement the field. Better make sure they work in the best possible way.